files/workflows/update_settings.yml

```
@@ -0,0 +1,34 @@
+name: Keep Settings Up To Date
+on:
+  schedule:
+    - cron:  '*/5 * * * *'
+
+jobs:
+  settings_update:
+    runs-on: ubuntu-latest
+    name: Create PR when settings are updated
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Submodule update --remote
+        run: |
+          git submodule update --init --remote
```

```suggestion
          git submodule update --init --remote --recursive
```

Adding `--recursive` will allow the `conjur-project-config` repo to use submodules (such as bash-lib) in the future without having to go back and update the repos that use it.  Adding it now prevents the need to release the change in stages later (first updating the workflow to use recursive, and then adding the submodule).